_posts/2017-07-19-runners-driver.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Concordion | Runners and Drivers"
+heading: "Concordion Presentations"
+description: "Layering test code with Runners and Drivers"
+identifier: "concordion-presentations"
+author: "Nigel Charman"
+homepage: http://tutansblog.blogspot.co.nz/
+---
+
+![Visual representation of Layering test code with Runners and Drivers](_posts/drivers-runners.png)
```

Comment:
Please trim the image so that the right and bottom borders are the same as the left and top. Also shrink it down by about a third and make it central across the page.